load-tests/ftp_pubsub/deployment/ingress.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ftp_pubsub
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+    - host: bal.perf.test
+      http:
+        paths:
+          - path: "/"
+            pathType: Prefix
+            backend:
+              service:
+                name: simple-producer
```

```suggestion
                name: simple-producer
```
Is this the correct service name?

---

Added the artifacts related to the k8s deployment of SFTP.